src/components/routing/routers/AppRouter.tsx

```diff
@@ -24,6 +25,7 @@ const AppRouter = () => {
           />
           <Route path={Paths.GAME} element={<Navigate to={Paths.LOBBY} />} />
           <Route path={Paths.CREATE_PLAYER} element={<CreatePlayer />} />
+          <Route path={`${Paths.JOINGAME}/:gameId`} element={<JoinGame />} />
```

Comment:
```suggestion
          <Route path={`${Paths.JOINGAME}/:currentGameId`} element={<JoinGame />} />
```
Then you can use the useCurrentGame hook, because it extracts the gameId which is called ":currentGameId"

Reply:
Wo soll ich die genau nutzen? Und was ist mein gain? (Stehe gerade auf dem schlauch)

Reply:
Lütfen Almanca Kullanmayalım. Şu anda nasıl hissediyorsanız, ben de öyle hissediyorum. :) Lütfen işleri kolaylaştıralım ve resmi dil olan İngilizceyi tercih edelim. Rica ediyorum :)

Reply:
@melihcatal İyi bir nokta. Herhangi bir belirsizlik varsa, bunları Almanca olarak yazmak daha hızlıdır ve elbette daha fazla İngilizce yazışma yapacağım. ;-)

Reply:
You don't have to extract the gameId in the component, the useCurrentGame hook extracts it for you and offers you the methods refreshGame (to get the game once) and startPollGame to get updates if the game changes on the server.

Reply:
@MorisCamporesi @BacLuc Thank you for your understanding :) --- Danke für Ihr Verständnis :) --- Anlayışınız için teşekkür ederim :)